### ISSUE TITLE
Ipa info fails since recent gem update 0.11.1

### ISSUE THREAD

**Issue by jedi4ever** [nomad/shenzhen#208]
On ruby 2.0.0 I'm now getting this error for a simple ipa info command.
Previous versions work fine.

````
$ ipa info some.ipa
invalid command. Use --help for more information
/Users/patrick/.rvm/rubies/ruby-2.0.0-p576/lib/ruby/site_ruby/2.0.0/rubygems/core_ext/kernel_require.rb:126:in `requir
      app_response = client.update_app_info({
                  ^
        from /Users/patrick/.rvm/rubies/ruby-2.0.0-p576/lib/ruby/site_ruby/2.0.0/rubygems/core_ext/kernel_require.rb:1
        from /Users/patrick/.rvm/gems/ruby-2.0.0-p576@ios/gems/shenzhen-0.11.1/lib/shenzhen/commands.rb:11
        from /Users/patrick/.rvm/rubies/ruby-2.0.0-p576/lib/ruby/site_ruby/2.0.0/rubygems/core_ext/kernel_require.rb:7
        from /Users/patrick/.rvm/rubies/ruby-2.0.0-p576/lib/ruby/site_ruby/2.0.0/rubygems/core_ext/kernel_require.rb:7
        from /Users/patrick/.rvm/gems/ruby-2.0.0-p576@ios/gems/shenzhen-0.11.1/bin/ipa:25:in `<top (requir
        from /Users/patrick/.rvm/gems/ruby-2.0.0-p576@ios/bin/ipa:23:in `load'
        from /Users/patrick/.rvm/gems/ruby-2.0.0-p576@ios/bin/ipa:23:in `<main>'
        from /Users/patrick/.rvm/gems/ruby-2.0.0-p576@ios/bin/ruby_executable_hooks:15:in `eval'
        from /Users/patrick/.rvm/gems/ruby-2.0.0-p576@ios/bin/ruby_executable_hooks:15:in `<main>'
````

**Comment by mattt** [comment]
Yanking 0.11.1. Fixed in 0.11.2. Apologies for any inconvenience.

**Comment by jedi4ever** [comment]
muchos gracias @mattt !
